site/app/libraries/database/DatabaseQueries.php

```diff
@@ -8075,7 +8075,7 @@ public function deleteQueue(string $queue_code) {
     public function getValidatedCode(string $queue_code, string $token = null) {
         //first check if the queue has an access code
         $this->course_db->query("SELECT * FROM queue_settings WHERE UPPER(TRIM(code)) = UPPER(TRIM(?)) AND open = true", [$queue_code]);
-        if ($this->course_db->getRowCount() > 0 && $this->course_db->row()['token'] == null) {
+        if ($this->course_db->getRowCount() > 0 && $this->course_db->rows()[0]['token'] == null) {
```

Comment:
Rather than changing this back. Make a tmp variable above this if statement which stores the result of `row()` and then use it both this if statement and line 8079.